src/Meshes/geoMesh.ts

```diff
@@ -0,0 +1,790 @@
+import { Vector3, TmpVectors } from "../Maths/math.vector";
+import { Scalar } from '../Maths/math.scalar';
+import { PHI, THIRDR3 } from '../Maths/math.constants';
+import { IsoVector } from '../Maths/math.isovector';
+
+
+
+/**
+ * Class representing data for one face OAB of an equilateral icosahedron
+ * When O is the isovector (0, 0), A is isovector (m, n)
+ * @hidden
+ */
+export class Primary {

```

Comment:
This will be exported as BABYLON.Primary on our UMD package. it feels a bit too abstract for an exported class. If it is an internal class (that is not expected to be used by users) add an underscore and name it a bit differently so that we will know what Primary means.

Reply:
Done

Reply:
Done